### ISSUE TITLE
content attribute empty in obsidian_markdown_notes

### ISSUE THREAD

**Issue by gentzeng** [sytone/obsidian-queryallthethings#19]
**Describe the bug**
I have some functionality where I display some parts of the content of markdown files with qatt. For that, I query `obsidian_markdown_notes` and the `content` attribute. It used to work but when I now log the results to the console, it seams like the the `content` is always just an empty string.

**To Reproduce**
1. Run the following query in obsidian
```qatt
postRenderFormat: html
​￼query: |
  SELECT *
  FROM obsidian_markdown_notes
​￼template: |
  {{log result}}
```
2. Check the results in the console.

**Expected behavior**
The results should contain the content of each note.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.4 LTS
 - Obsidian Version: 1.6.5
 - Plugin Version 1.0.1
 
 Looks like in commit `826188605254d01a1afc5a5188b066754ae90fef` in `src/note.ts`, content is just being set to `''` for some reason, maybe debugging.

**Comment by sytone** [comment]
Thanks for creating an issue, this is corrected in 1.0.2. It was emptied as I was trying to use the IndexDB for caching and it was too large. I neglected to set it back. Apologies for impacting on your workflows. 

As part of this I also corrected an issue with the upsert of tasks and lists when a note is changed.

Documentation has been updated for the new tables as well. If you use `obsidian_notes` it also has a cleaner approach in the SQL queries. Longer term I will eventually remove the ones with markdown in them to keep it consistent. There will be warnings for this.

**Comment by sytone** [comment]
Release https://github.com/sytone/obsidian-queryallthethings/releases/tag/1.0.2 is now available.

**Comment by gentzeng** [comment]
Thank you so much for quickly fixing this :)

**Comment by gentzeng** [comment]
If you deprecate `obsidian_markdown_notes` will there be another way to query the content of the notes? AFAIK there is no other plugin that allows that

**Comment by sytone** [comment]
Technical details in coming...

TLDR: `obsidian_notes` contains the same data as `obsidian_markdown_notes`, `obsidian_notes` will get improvements over time, `obsidian_markdown_notes` will not.

The table `obsidian_notes` contains the same data as `obsidian_markdown_notes`. I am renaming the tables as I use a regex replace in the query for `obsidian_markdown_notes` and with `obsidian_notes` it is actually a in memory table and not a referenced object array.

The referencing of the object array while working dose cause some issues with more complex queries. This is another reason why I will depreciate it.

**Comment by sytone** [comment]
@gentzeng following up on this issue, is the resolution and information clear?

**Comment by gentzeng** [comment]
@sytone Yes, `obsidian_note` works perfectly fine :+1:

**Comment by sytone** [comment]
Awesome to hear!
